Fortinet/fortigate/_meta/fields.yml

```
@@ -0,0 +1,14 @@
+icmp.request.type:
+  name: icmp.request.type
+  type: keyword
+  description: ""
+
+icmp.request.code:
+  name: icmp.request.code
+  type: keyword
+  description: ""
+
+rule.apprisk:
+  name: rule.apprisk
+  type: keyword
+  description: ""
```

```suggestion
fortinet.apprisk:
  fortinet: rule.apprisk
  type: keyword
  description: "Risk level of the application."
```

According to this [documentation](https://docs.fortinet.com/document/fortigate/6.2.3/fortios-log-message-reference/357866/log-message-fields) this is the `Risk level of the application.` which is not an ECS field and not a `rule.` field